seqr/utils/file_utils_tests.py

```diff
@@ -30,34 +30,3 @@ def test_mv_file_to_gs(self, mock_logger, mock_subproc):
         mock_subproc.Popen.assert_called_with('gsutil mv /temp_path gs://bucket/target_path', stdout=mock_subproc.PIPE, stderr=mock_subproc.STDOUT, shell=True)
         mock_logger.info.assert_called_with('==> gsutil mv /temp_path gs://bucket/target_path', None)
         process.wait.assert_called_with()
-
-    @mock.patch('seqr.utils.file_utils.subprocess')
-    @mock.patch('seqr.utils.file_utils.logger')
-    def test_get_gs_file_list(self, mock_logger, mock_subproc):
-        with self.assertRaises(Exception) as ee:
-            get_gs_file_list('/temp_path')
-        self.assertEqual(str(ee.exception),  'A Google Storage path is expected.')
-
-        process = mock_subproc.Popen.return_value
-        process.wait.return_value = -1
-        process.stdout = [b'-bash: gsutil: command not found.\n', b'Please check the path.\n']
-        with self.assertRaises(Exception) as ee:
-            get_gs_file_list('gs://bucket/target_path/', user=None)
-        self.assertEqual(str(ee.exception), 'Run command failed: -bash: gsutil: command not found. Please check the path.')
-        mock_subproc.Popen.assert_called_with('gsutil ls gs://bucket/target_path', stdout=mock_subproc.PIPE,
-                                              stderr=mock_subproc.STDOUT, shell=True)
-        mock_logger.info.assert_called_with('==> gsutil ls gs://bucket/target_path', None)
-        process.wait.assert_called_with()
-
-        mock_subproc.reset_mock()
-        mock_logger.reset_mock()
-        process.wait.return_value = 0
-        process.stdout = [b'\n', b'\n', b'Updates are available for some Cloud SDK components.  To install them,\n',
```

Comment:
the case where gsutil is printing warnings is no longer tested

Reply:
In one of the `get_anvil_vcf_list` tests, the mocked stdout has a warning as the code below. The test covers this case.
```
        mock_subprocess.return_value.stdout = [
            b'Warning: some packages are out of date',
            b'gs://test_bucket/test.vcf', b'gs://test_bucket/data/test.vcf.gz', b'gs://test_bucket/test.tsv',
        ]
```